src/lib/s3.js

```diff
@@ -37,27 +37,29 @@ export const saveFileToS3 = (Key, Body) => {
 };
 
 export const saveProfile = async (profileId, repos) => {
-  const metadata = { objectKeys: [] };
-  for (const repo of repos) {
-    if (repo.files) {
-      const objectKeys = await saveProfileFiles(profileId, repo);
-      metadata.objectKeys = [...metadata.objectKeys, ...objectKeys];
-    }
-  }
-  const key = `${profileId}/dependencies.json`;
-  const savedFile = await saveFileToS3(key, metadata);
-  return savedFile.Key.split('/')[0];
+  const objectKeys = await Promise.filter(repos, repo => repo.files, {
+    concurrency: 10,
+  })
```

Comment:
I don't see the purpose of the Promise.filter or the concurrency. What is it for?

Reply:
I'm just being defensive with the filter. The concurrency is to speed up the process.

Reply:
It doesn't make sense, there almost can't be something faster than the filtering here. The `Promise` + `concurrency`  is just bloating the code.